src/platform/packages/shared/kbn-monaco/src/languages/xjson/lexer_rules/xjson.ts

```diff
@@ -117,10 +117,12 @@ export const languageConfiguration: monaco.languages.LanguageConfiguration = {
   brackets: [
     ['{', '}'],
     ['[', ']'],
+    ['"""', '"""\n'],
```

Comment:
Little surprising to see `\n` included here as I don't believe newlines are required by the grammar. Also, does this still work with the edge case of having `\""""`?